### ISSUE TITLE
[Opinion] All-true-ation * is hard to read

### ISSUE THREAD

**Issue by chrismilson** [satwikkansal/wtfpython#255]
The explanation in [All-true-ation *](https://github.com/satwikkansal/wtfpython#-all-true-ation-)

> `all([[]])` returns `False` because `not []` is `True` is equivalent to `not False` as the list inside the iterable is empty.

Feels like verbal diarrhoea to me. Perhaps something like:

> `all([[]])` returns `False` because its single element, `[]`, is an empty list, which is falsy in python.

The second point is also a bit hard to follow; the use of `not` is difficult to understand on a first pass. I would suggest:

> `all([[[]]])` and higher recursive variants are always `True` since the inner array, `[[...]]`, has one element, and lists with values are truthy.

I am willing to make a PR for these changes. Just let me know.

**Comment by satwikkansal** [comment]
I think you're right, it's confusing to follow now that I think about it, would appreciate the PR if you have time :) 

Thanks @chrismilson
